modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -668,40 +667,42 @@ TensorMechanicsAction::verifyOrderAndFamilyOutputs()
     mooseError("The number of families assigned to material outputs must be: 0 to be assigned "
                "MONOMIAL; 1 to assign all outputs the same value, or the same size as the number "
                "of generate outputs listed.");
-  // Make sure all outputs are standard constant value
-  if (order_check.size() == 0)
-  {
-    order_display_notice = true;
-    _material_output_order.assign(_generate_output.size(), "CONSTANT");
-  }
-  if (family_check.size() == 0)
-  {
-    family_display_notice = true;
-    _material_output_family.assign(_generate_output.size(), "MONOMIAL");
-  }
 
-  // For only one order, make all orders the same magnitude
-  if (order_check.size() == 1)
+  if (order_check.size() == _generate_output.size())
   {
-    order_display_notice = true;
-    _material_output_order.assign(_generate_output.size(), _material_output_order[0]);
+    for (const auto & out : order_check)
+      _material_output_order.push_back(out);
   }
-  if (family_check.size() == 1)
+  else
   {
-    family_display_notice = true;
-    _material_output_family.assign(_generate_output.size(), _material_output_family[0]);
-  }
-
-  if (order_display_notice)
+    if (order_check.size() == 0)
+      // Make sure all outputs are standard constant value
+      _material_output_order.assign(_generate_output.size(), "CONSTANT");
+    // For only one order, make all orders the same magnitude
+    if (order_check.size() == 1)
+      _material_output_order.assign(_generate_output.size(), _material_output_order[0]);
     Moose::out << COLOR_CYAN << "*** Automatic applied material output orders ***"
```

Comment:
Can we make this action less chatty?

Reply:
Like suppress the orders and families constructed?

Reply:
Yeah, how about we add a `verbose` bool parameter that defaults to `false`.  (or maybe call it `debug`, IDK)

Reply:
It would be nice to have a standardized name for this type of toggle.

Reply:
So you could run `./mooseapp-opt -i input.i verbose=true` or `./mooseapp-opt -i input.i debug=true`